ivi-layermanagement-examples/EGLWLInputEventExample/src/WLContext.cpp

```diff
@@ -85,7 +85,7 @@ create_cursors(WLContext* wlContext)
 		return;
 	}
 	wlContext->SetWLCursor(
-			(wl_cursor*) malloc(sizeof wlContext->GetWLCursor()));
+			(wl_cursor*) malloc(sizeof *wlContext->GetWLCursor()));
```

Comment:
sizeof and * has same precedence but presumably this is correct and unambiguous, despite that some parentheses might help.

But it's still not 100% obvious just reading it.  This code presumably assumes that GetWLCursor returns the same type (same size) as SetWLCursor expects as input?

1. If that type is indeed a pointer to wl_cursor, would not sizeof(wl_cursor) be as easy to understand here?
  
Overall it looks a bit strange.  There is a call to "wlContext->GetWLCursor()" as a side effect of the expression here?  Or does that evaluate to a constant expression...?

2. I haven't checked SetWLCursor - just asking here.  malloc returns uninitialized memory, right?  Is that OK as input to SetWLCursor?  (Only care about storing the pointer to the storage space, not the value)?  

If this is all fine, I'd recommend a comment explaining why the line is correct.

Reply:
Thank you for review. I will fix this issue and send an updated patch